Automattic/jetpack
# Payments Block: only add PayPal email to Payment CPT.

#### Changes proposed in this Pull Request:

We may not need to add the PayPal email address meta field to all post types, since we only need it for the Payment block.

The meta were added in #10402, and restricted to `jp_pay_product`, except for that email field.

#### Other information:


#### Jetpack product discussion

> **Note**
> See discussion at p1655881305185899-slack-CJS75TX3R

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site using a paid plan.
* Go to Posts > Add New
* Add a Pay with PayPal block
* fill information about your product, and publish your post.
* You should be able to click on the button to buy the product.
* If you note that post ID, when going to `yoursite.com/wp-json/wp/v2/posts/$$post-id$$`, you should not see the `spay_email` field in `meta`.
